activerecord/test/cases/bind_parameter_test.rb

```diff
@@ -238,10 +238,11 @@ def assert_bind_params_to_sql
           #
           #   SELECT `authors`.* FROM `authors` WHERE `authors`.`id` IN (1, 2, 3)
           #
-          if current_adapter?(:Mysql2Adapter)
-            params = bind_params((1..3).map(&:to_s))
+          params = if current_adapter?(:Mysql2Adapter) || current_adapter?(:TrilogyAdapter)
```

Comment:
```suggestion
          params = if current_adapter?(:Mysql2Adapter, :TrilogyAdapter)
```

(definitely not important 😅)